dev-util/webstorm/webstorm-2017.2.2.172.3757.55.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils versionator
+
+SLOT="$(get_major_version)"
+MY_PV="$(get_version_component_range 4-6)"
+MY_PN="WebStorm"
```

Comment:
I don't like this variable polka. MY_PV is used exactly once, yet you make it variable while the version part from SRC_URI is inlined. Decide on one — either have both in variables, or both inline. Whichever way you go, also include `WebStorm-` part in both since MY_PN is only used with the versions.

Reply:
fixed